src/containers/index.ts

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux'
+import { login, logout } from '../actions';
+
+const mapStateToProps = (state) => ({
+  isAuthenticated: state.authenticate.isAuthenticated
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  loginSucceed: () => dispatch(login(true)),
```

Comment:
`loginSucceeded` to be equivalent with `loginFailed`

Reply:
Changed it to loginSucceeded